packages/README.md

```
@@ -0,0 +1,129 @@
+# Superfunctions Packages
+
+Shared packages for the superfunctions ecosystem - both TypeScript and Python.
+
+## Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)